src/types/missing_scene.ts

```diff
@@ -13,23 +8,63 @@ export default class MissingScene {
 }
 
 import { sceneCollection, missingSceneCollection } from "../database/index";
+import Marker from "../types/marker";
+import { removeSceneFromQueue } from "../queue/processing";
+import LabelledItem from "../types/labelled_item";
+import ActorReference from "../types/actor_reference";
+import MovieScene from "../types/movie_scene";
+import Image from "../types/image";
+import { index as sceneIndex } from "../search/scene";
 import * as logger from "../logger";
 export async function purgeMissingScenes() {
   const items = await missingSceneCollection.getAll();
-  items.forEach(async item => {
+  for (const [key, item] of items.entries()) {
```

Comment:
In this situation, `items` is an array `MissingScene[]` and not a Map like before, so does not have a `entries()` method. Use a simple array for/of iteration

```suggestion
  for (const missingScene of items) {
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/entries
?

though regardless, your suggestion is cleaner since I don't actually need the key.

Reply:
Yeah that's my bad, sorry